<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-52C1ANV" failures="2" tests="4" name="testCases.LoginTest" time="22.092" errors="0" timestamp="2022-07-05T20:26:44 AMT">
  <testcase classname="testCases.LoginTest" name="TestSuccess" time="0.376"/>
  <testcase classname="testCases.LoginTest" name="testFail" time="0.448">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.LoginTest.testFail(LoginTest.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testFail -->
  <testcase classname="testCases.LoginTest" name="testSkipp" time="0.003">
    <skipped/>
  </testcase> <!-- testSkipp -->
  <testcase classname="testCases.LoginTest" name="loginPopUpGUI_Test" time="0.55">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Log In     ] but found [Log In]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Log In     ] but found [Log In]
at testCases.LoginTest.loginPopUpGUI_Test(LoginTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginPopUpGUI_Test -->
</testsuite> <!-- testCases.LoginTest -->
